### ISSUE TITLE
Document how to use outputs

### ISSUE THREAD

**Issue by raytio** [terraform-aws-modules/terraform-aws-rds-aurora#226]
## Is your request related to a new offering from AWS?
No

## Is your request related to a problem? Please describe.
I am unable to refer to the outputs variables from this module. For example I have the following module definition:
```
module "db" {
  source  = "terraform-aws-modules/rds-aurora/aws"
  version = "~> 3.0"

  name              = "${var.env}-postgresql-transactions"
  engine            = "aurora-postgresql"
  engine_mode       = "serverless"
  storage_encrypted = true
  username          = local.db_creds.username
  password          = local.db_creds.password
  vpc_id                = data.aws_ssm_parameter.vpc_id.value
  subnets               = split(",", data.aws_ssm_parameter.vpc_private_subnets.value)
  create_security_group = true

  replica_scale_enabled = false
  replica_count         = 0

  monitoring_interval = 60

  apply_immediately   = true
  skip_final_snapshot = true

  db_parameter_group_name         = aws_db_parameter_group.postgresql_transactions.id
  db_cluster_parameter_group_name = aws_rds_cluster_parameter_group.postgresql_transactions.id

  scaling_configuration = {
    auto_pause               = true
    min_capacity             = 2
    max_capacity             = 16
    seconds_until_auto_pause = 300
    timeout_action           = "ForceApplyCapacityChange"
  }
}
```
According to the documentation, one of the outputs is `rds_cluster_arn`.  When I attempt to refer to that I get an error:

```
An output value with the name "rds_cluster_arn" has not been declared in
module.main.module.db.
```

## Describe the solution you'd like.
Could the README give an example of how the outputs could be used?

## Describe alternatives you've considered.
The research I've done indicates that what I'm doing should work so I suspect there's something very simple that I'm doing wrong.

**Comment by bryantbiggs** [comment]
examples are provided which demonstrate this already https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/blob/master/examples/postgresql/outputs.tf

**Comment by raytio** [comment]
Thanks for the prompt response, however I'm still missing something fundamental I think because I get:
```
An output value with the name "rds_cluster_arn" has not been declared in
module.main.module.db.
```

The module and output definition:
```
module "db" {
  source  = "terraform-aws-modules/rds-aurora/aws"
  version = "~> 3.0"

  name              = "${var.env}-postgresql-transactions"
  engine            = "aurora-postgresql"
  engine_mode       = "serverless"
  storage_encrypted = true
  username          = local.db_creds.username
  password          = local.db_creds.password
  vpc_id                = data.aws_ssm_parameter.vpc_id.value
  subnets               = split(",", data.aws_ssm_parameter.vpc_private_subnets.value)
  create_security_group = true

  replica_scale_enabled = false
  replica_count         = 0

  monitoring_interval = 60

  apply_immediately   = true
  skip_final_snapshot = true

  db_parameter_group_name         = aws_db_parameter_group.postgresql_transactions.id
  db_cluster_parameter_group_name = aws_rds_cluster_parameter_group.postgresql_transactions.id

  scaling_configuration = {
    auto_pause               = true
    min_capacity             = 2
    max_capacity             = 16
    seconds_until_auto_pause = 300
    timeout_action           = "ForceApplyCapacityChange"
  }
}

# aws_rds_cluster
output "postgresql_rds_cluster_arn" {
  description = "The ARN of the cluster"
  value       = module.db.rds_cluster_arn
}

output "postgresql_rds_cluster_db_name" {
  description = "The DB name of the cluster"
  value       = module.db.rds_cluster_database_name
}
```

**Comment by bryantbiggs** [comment]
is this your entire config? the error message `module.main.module.db` makes me believe you are wrapping the RDS Aurora module inside another module called `main`. if that is the case, you will have to export the value from your `main` module and then reference that modules output in order to get the ARN

**Comment by raytio** [comment]
Many thanks for your response. In the end I just created the resources directly which was easy enough in my case. In case it helps someone:
```
// Firstly we will create a random generated password which we will use in secrets.
resource "random_password" "password" {
  length           = 16
  special          = true
  override_special = "_%@"
}

// Secretmanager
// Now create secret and secret versions for database master account
resource "aws_secretsmanager_secret" "secretmasterDB" {
   name = "Masteraccoundb"
}

resource "aws_secretsmanager_secret_version" "version" {
  secret_id = aws_secretsmanager_secret.secretmasterDB.id
  secret_string = <<EOF
   {
    "username": "root",
    "password": "${random_password.password.result}"
   }
EOF
}

# Lets import the Secrets which got created recently and store it so that we can use later.
data "aws_secretsmanager_secret" "secretmasterDB" {
  arn = aws_secretsmanager_secret.secretmasterDB.arn
}

data "aws_secretsmanager_secret_version" "creds" {
  secret_id = data.aws_secretsmanager_secret.secretmasterDB.arn
}

# After Importing the secrets Storing the Imported Secrets into Locals

locals {
  db_creds = jsondecode(
  data.aws_secretsmanager_secret_version.creds.secret_string
   )
}

// RDS
resource "aws_db_subnet_group" "default" {
  name = "${var.env}-postgresql-transactions-cluster-subnet-group"
  subnet_ids = split(",", data.aws_ssm_parameter.vpc_private_subnets.value)
}

resource "aws_rds_cluster" "transactions" {
  cluster_identifier              = "${var.env}-postgresql-transactions-cluster"
  database_name              = "${var.env}PostgresqlTransactionsDB"
  engine            = "aurora-postgresql"
  engine_mode       = "serverless"
  storage_encrypted = true
  master_username          = local.db_creds.username
  master_password          = local.db_creds.password
  apply_immediately   = true
  enable_http_endpoint = true
  db_cluster_parameter_group_name = aws_rds_cluster_parameter_group.postgresql_transactions.id
  db_subnet_group_name = aws_db_subnet_group.default.name
  port = 5432
  kms_key_id = aws_kms_key.encrypt_decrypt.arn
  skip_final_snapshot = true
  vpc_security_group_ids = [
    aws_security_group.lambda_ongdb.id
  ]
  scaling_configuration {
    auto_pause               = true
    min_capacity             = 2
    max_capacity             = 16
    seconds_until_auto_pause = 300
    timeout_action           = "ForceApplyCapacityChange"
  }
}

resource "aws_rds_cluster_parameter_group" "postgresql_transactions" {
  name        = "${var.env}-aurora-postgres-cluster-parameter-group"
  family      = "aurora-postgresql10"
  description = "${var.env}-aurora-postgres-cluster-parameter-group"
}
```

**Comment by barundel** [comment]
Just an FYI for anyone else that reaches this issue. I had the same problem above and realised that I had copied one of the examples where the version was tied to 3.0.0 obviously the current version is 5.0+ and the value of the outputs have been changed. 

in 3.0 its module.db.this_rds_cluster_database_name
in 50.+ its module.db.rds_cluster_database_name

so the this_ was removed. I bumped my version to the latest and the outputs referenced in the readme and outputs.tf worked for me.
